arch/arm/dts/kent_gfxdemo.dts

```diff
@@ -0,0 +1,121 @@
+/*
+ * kent_gfxdemo.dts - Device Tree file for Kent Scientific KSC2024 Graphics Demo
+ *
+ *  Copyright (C) 2017 Microchip Corporation
+ *                     Wenyou Yang <wenyou.yang@microchip.com>
+ *  Copyright (C) 2023 Innovative Electronics, LLC
+ *                     Sam Birch <sbirch@innovative-electronics.com>
+ *
+ * This file is dual-licensed: you can use it either under the terms
+ * of the GPL or the X11 license, at your option. Note that this dual
+ * licensing only applies to this file, and not this project as a
+ * whole.
+ *
+ *  a) This file is free software; you can redistribute it and/or
+ *     modify it under the terms of the GNU General Public License as
+ *     published by the Free Software Foundation; either version 2 of the
+ *     License, or (at your option) any later version.
+ *
+ *     This file is distributed in the hope that it will be useful,
+ *     but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *     MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *     GNU General Public License for more details.
+ *
+ * Or, alternatively,
+ *
+ *  b) Permission is hereby granted, free of charge, to any person
+ *     obtaining a copy of this software and associated documentation
+ *     files (the "Software"), to deal in the Software without
+ *     restriction, including without limitation the rights to use,
+ *     copy, modify, merge, publish, distribute, sublicense, and/or
+ *     sell copies of the Software, and to permit persons to whom the
+ *     Software is furnished to do so, subject to the following
+ *     conditions:
+ *
+ *     The above copyright notice and this permission notice shall be
+ *     included in all copies or substantial portions of the Software.
+ *
+ *     THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+ *     EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES
+ *     OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+ *     NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+ *     HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY,
+ *     WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+ *     FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+ *     OTHER DEALINGS IN THE SOFTWARE.
+ */
+/dts-v1/;
+#include <dt-bindings/gpio/gpio.h>
+#include "sama5d27_som1.dtsi"
+#ifdef CONFIG_CLK_SCMI
+#include "sama5d2-optee.dtsi"
+#endif
+
+/ {
+	model = "Kent Scientific KSC2024 Graphics Demo";
+	compatible = "atmel,kent-gfxdemo", "atmel,sama5d2", "atmel,sama5";
+
+	chosen {
+		bootph-all;
```

Comment:
I don't remember needing `bootph-all` being a part of my life last time I did this. That doesn't mean you don't need it -- is it something you got from copying the ek1 dts?

Reply:
Yes, it is, and I haven't tested without it. Shall I do it now, or shall we put it in the backlog?

Reply:
Nah, I'm not asking for a change here, I'm just curious if you knew anything about it.